tests_backend/tests/backend/updates_test.ts

```diff
@@ -29,17 +34,18 @@ const baseDataIos = {
   is_prod: true,
 }
 
-function getBaseData(): typeof baseData {
-  return structuredClone(baseData)
-}
-
 function getBaseDataIos(): typeof baseData {
   return structuredClone(baseDataIos)
 }
 
 const noNew = { message: 'No new version available' }
 
-export async function testUpdateEndpoint(backendBaseUrl: URL, supabase: SupabaseType) {
+async function perapreUpdateTest(_backendBaseUrl: URL, supabase: SupabaseType) {
```

Comment:
```suggestion
async function prepapreUpdateTest(_backendBaseUrl: URL, supabase: SupabaseType) {
```

Reply:
I will not commit this thru git (as you have not changed this in the test declaration) but I fixed this in the latest commit